app/src/main/java/app/accrescent/client/receivers/AppInstallBroadcastReceiver.kt

```diff
@@ -32,7 +34,7 @@ class AppInstallBroadcastReceiver : BroadcastReceiver() {
         val packageName = intent.getCharSequenceExtra(PackageInstaller.EXTRA_PACKAGE_NAME).toString()
         val appName = runBlocking { repoDataRepository.getApp(packageName) }?.name
 
-        val notificationManager = NotificationManagerCompat.from(context)
+        val notificationManager = context.getSystemService<NotificationManager>()!!
```

Comment:
Why change how we grab a handle to the notification manager?

Reply:
NotificationManagerCompat is only needed on SDK 20 and below. Since we don't support these SDKs, using NotificationManager is the better option.

Reply:
Gotcha, thanks. Maybe next time just split that out into a separate commit since it's unrelated. You changed from the `getSystemService` approach to `NotificationManagerCompat` in https://github.com/accrescent/accrescent/commit/f51c7c3722baa3972ecedd4bdc49ba50d95eb19b, in which case I assumed there was some functionality exclusive to `NotificationManagerCompat` being used, which was why I found changing it back to be odd.

Reply:
Sure, will do that next time.